recipe-app.xcodeproj/xcuserdata/arthur.hermann.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This shouldn't be on your PR. Let's create a .gitignore so this stops happening.